src/babel-serializer/file.hh

```diff
@@ -59,4 +75,95 @@ struct file_output_stream
 private:
     FILE* _file = nullptr;
 };
+
+namespace detail
+{
+struct mmap_info
+{
+#if defined(CC_OS_WINDOWS)
+    void* file_handle = nullptr;
+    void* file_mapping_handle = nullptr;
+#else
```

Comment:
You can use cc native/win32_fwd.hh to have access to `HANDLE` etc without dragging the headers in

Reply:
Done!